encrypter.cpp

```diff
@@ -0,0 +1,168 @@
+#include <iostream>
+#include <fstream>
+#include <regex>
+#include <string>
+#include <stdlib.h>
+#include <unistd.h>
+
+using namespace std;
+
+string read_file(string& address, string error_message, bool clean = false)
+{
+  ifstream f;
+
+  f.open(address);
+  if(!f)
+  {
+    cout << error_message;
```

Comment:
It's better to make a function like put_error and incapsulate print an error and exit in this function.